docs/settingup.md

```diff
@@ -1165,6 +1165,86 @@ Unpublish sheets in the current app.
 }
 ```
 
+</details><details>
+<summary>subscribeobjects</summary>
+
+## Subscribeobjects action
+
+Subscribe to any object in the currently active app.
+
+### Settings
+
+* `clear`: Clear any previously subscribed objects from subscription list.
+* `ids`: List of object ID's to subscribe to.
+
+### Example
+
+Subscribe to two objects in the currently active app, unsubscribing to any previous subscriptions. 
```

Comment:
Suggestion:
"Subscribe to two objects in the currently active app and remove any previous subscriptions."